packages/gatsby-theme-newrelic/src/components/Link.js

```diff
@@ -31,6 +32,29 @@ const Link = ({ to, onClick, ...props }) => {
     }
   `);
 
+  const properties =
+    typeof instrumentation === 'string'
+      ? { customProp: instrumentation }
```

Comment:
Looking at this more, I'm not so sure I like this. The `customProp` name feels a bit arbitrary, and not super useful. For example, if you defined this as `<Link instrumentation="GlobalHeader" />`, what would you expect? By allowing a string, it's not very clear what the intent is here.

I think we should enforce an object so that the additional instrumented properties are very clearly defined. Let's not allow a string as it could lead to more confusion than its worth.

Reply:
I was more thinking for the purposes of content contributors wanted to add instrumentation, which is why i thought maybe a string might be easier to create rather than an object at times! 

I agree it's definitely arbitrary, but one of the requests is that sign up links could be tracked wherever they occur and most of the time that should be configured by docs writers. I'm totally for enforcing the object, just will have to make it clear to content contributors that this is the way we would like it configured! :)

Reply:
Absolutely! I think its reasonable to ask that they define this as an object so that properties are named accordingly. I just think `customProp` is going to get lost, or is meaningless in this case. Let's do what we can to teach content contributors how to add instrumentation when needed 🙂